src/message.rs

```diff
@@ -1384,6 +1391,67 @@ pub async fn mdn_from_ext(
     None
 }
 
+pub(crate) async fn ndn_from_ext(
```

Comment:
I like doc comments, especially for new functions ;)

Reply:
Done.